### ISSUE TITLE
Allow pip option passthrough

### ISSUE THREAD

**Issue by sean-abbott** [mitsuhiko/pipsi#83]
Which would, by default, give you things like "different python intepretors"

My system default is python2, but I'd like to be able to install a package with python3.

**Comment by scolby33** [comment]
I would like to +1 this suggestion as well. I've had to do some manual "pipsi" installs to install some packages that require Python 2.

**Comment by ryneeverett** [comment]
I'm all in favor of pip option pass-through, but isn't the use case you're asking about provided by the `pipsi install --python` flag?

**Comment by scolby33** [comment]
I didn't know that existed! Thanks for pointing it out. I've tried `pipsi help` in the past, but never tried `pipsi install -h`...

**Comment by pjz** [comment]
I think this suggestion is still valid, as it keeps pipsi from having to understand pip commands.  I think it will save a lot of work to let pip do pip stuff and let pipsi just make sure that the pip in question is one in the per-script virtualenv.  Oh, and it should link to any files in the virtualenv/bin that weren't there before pip was run - not try to figure out what they'll be, but rather observe what they *are*.

**Comment by bitprophet** [comment]
FWIW this seems like a duplicate (if a generalization) of #15.

**Comment by scolby33** [comment]
I agree with @pjz on the linking, too. That would have saved a lot of trouble on another issue I had with pipsi, installing jupyter, which uses some sort of meta-package and doesn't appear to pipsi to have any CLI entry points.
